sys/picolibc_syscalls_default/syscalls.c

```diff
@@ -24,6 +24,7 @@
 #include <stdio.h>
 #include <sys/times.h>
 
+#include "irq_arch.h"
```

Comment:
This seems unrelated. Can you explain ?

Reply:
Oh I already moved that to a different PR #14901

But this PR will be obsolete with #14895 anyway.

Reply:
Hmm ok, and #14895 should be merged before Y2038 ;)